### PR TITLE
Error on non-existing translations.

### PR DESCRIPTION
**Pull request by ecraven**
Add missing token PASSENGER_CABIN to equipment-core.

Use l:get from now on to get the translation or nil (no error if an entry
doesn't exist)

If a translation doesn't exist, the missing token and the resource and language
code it was supposed to be in are shown:

error: [string "[T] @modules/CrewContracts/CrewContracts.lua"]:402: missing translation token: `NAVIGATOR' in resource `module-crewcontracts', language code `en'.
stack traceback:
	[C]: in function '__index'
	[string "[T] @modules/CrewContracts/CrewContracts.lua"]:402: in function 'cb'
	[string "[T] @libs/Event.lua"]:34: in function '?'
	[string "[T] @libs/Event.lua"]:185: in function <[string "[T] @libs/Event.lua"]:180>

### CODE REVIEW

**Comment by ecraven** [comment]
This would ensure that we actually catch missing translation tokens. If you really need to check for the existence of a token, use l:get("FOO") instead of l.FOO.

File: src/LuaLang.cpp [hunk 1]
```diff
@@ -10,7 +10,27 @@
 
 static int _resource_index(lua_State *l)
 {
-	return luaL_error(l, "unknown translation token: %s", lua_tostring(l, 2));
+	std::string token = lua_tostring(l, 2);
```
Comment (ecraven) [inline src/LuaLang.cpp]: fixed and force pushed

**Comment by laarmen** [comment]
@impaktor do you wish to test this? Otherwise, I'd say it's good to go.

**Comment by impaktor** [comment]
It someone _wants_ me to test it I can do it tomorrow, otherwise, I have no opinions. Merge when ready.

**Pull request merged**